### DISCUSSION TITLE
How to resume from checkpoint without load optimiser

### DISCUSSION THREAD

**Discussion by yikuanli** [Lightning-AI/pytorch-lightning#6562] [code help: CV]
I have a pre-training model, while I just want to load the model and train it with a different optimiser, I don't know how to resume from the checkpoint without load the optimiser using pytorch_lightning

**Comment by hiyyg** [comment]
I think you need to manually remove the solver and scheduler states, like https://github.com/aim-uofa/AdelaiDet/blob/master/tools/remove_optim_from_ckpt.py.
